client/hbs/editor.hbs

```diff
@@ -2,7 +2,7 @@
 	{{#if user}}
 	{{set this "admin" true}}
 	{{> head.hbs}}
-	<body>
+	<body class="static">
```

Comment:
I'd instead add class `.editor` to the `<html>` tag and style based off of that.